tweet_on_upcoming_and_recently_released_movies_using_azure_openai/README.md

```diff
@@ -0,0 +1,40 @@
+# Tweet on upcoming and recently released movies using Azure OpenAI
+
+## Use Case
+Obtaines information on few upcoming and recently released movies from [TMDB](https://www.themoviedb.org/), generates a creative tweet using Azure OpenAI GPT3 models, and tweets it out.  
```

Comment:
```suggestion
Obtain information on a few upcoming and recently released movies from [TMDB](https://www.themoviedb.org/), generate a creative tweet using Azure OpenAI GPT3 models, and tweet it out.  
```

Reply:
Addressed in https://github.com/ballerina-guides/ai-samples/pull/63/commits/f7ce252feadcc1c2f7f19f7b18a3f078cbe22927.